build/__phpVersions.sh

```diff
@@ -1,15 +1,15 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-PHP_BUILD_BASE_TAG='20191205.1'
-PHP_RUNTIME_BASE_TAG='20191205.1'
-COMPOSER_VERSION='1.9.0'
-COMPOSER_SHA384='a5c698ffe4b8e849a443b120cd5ba38043260d5c4023dbf93e1558871f1f07f58274fc6f4c93bcfd858c6bd0775cd8d1'
-PHP73_VERSION='7.3.11'
+PHP_BUILD_BASE_TAG='20191218.4'
+PHP_RUNTIME_BASE_TAG='20191218.4'
+COMPOSER_VERSION='1.9.1'
+COMPOSER_SHA384='baf1608c33254d00611ac1705c1d9958c817a1a33bce370c0595974b342601bd80b92a3f46067da89e3b06bff421f182'
+PHP73_VERSION='7.3.12'
```

Comment:
Hmm, we must add 7.4.1 version in future

Reply:
thanks adding an workitem for this ..